### ISSUE TITLE
Movies list is empty on HTC Desire

### ISSUE THREAD

**Issue by GoogleCodeExporter** [peteaky/android-xbmcremote#451]
```
What steps will reproduce the problem?
1. Select TV Shows. The shows list properly with banners and show all episodes. 
2. Select Movies. No movies appear in the list. The message No movies found is 
shown.
3. Select Movie Actors. The list is populated with actors.
4. Select an actor. No movies appear in the list. The message No movies found 
is shown.

What is the expected output? What do you see instead?
I expect that selecting Movies will result in the list of movies in my library 
displaying on screen. Note that I do have several hundred movies in my library 
that I am able to browse and select in XBMC.

What version of XBMC Remote are you using?
v0.8.4-beta1. Revision 703


Which Android device are you using (and which ROM, if custom)?
HTC Desire from Telus (Canada) with Android 2.2.


Which XBMC revision are you running and on which platform?
XBMC 10.0 (Dharma) r35648 on Apple TV installed with patchstick with a Broadcom 
card installed.

Please provide any additional information below.
The remove is working well for TV Shows. The issue is strictly with movies.

```

Original issue reported on code.google.com by `lman...@gmail.com` on 22 Dec 2010 at 7:00

**Comment by GoogleCodeExporter** [comment]
```
Are you using a central mysql database with XBMC? If not I'd need more detailed 
info, such as a logfile.
```

Original comment by `phree...@gmail.com` on 16 Jan 2011 at 3:37

**Comment by GoogleCodeExporter** [comment]
```
I am using a central mysql database. Thanks.
```

Original comment by `lman...@gmail.com` on 16 Jan 2011 at 5:34

**Comment by GoogleCodeExporter** [comment]
```
Alright this should be fixed then in r394.
```

Original comment by `phree...@gmail.com` on 17 Jan 2011 at 7:53
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I have verified this fix with XBMC Remote v0.8.4 r717. Thanks for the fix.

http://code.google.com/p/android-xbmcremote/downloads/detail?name=XBMC_Remote_v0
.8.4_r717_signed.apk&can=1&q=
```

Original comment by `lman...@gmail.com` on 19 Jan 2011 at 7:46
